include/deal.II/non_matching/mapping_info.h

```diff
@@ -96,10 +96,10 @@ namespace NonMatching
      * with the "unfiltered" index. The default argument for
      * @p n_unfiltered_cells disables this built-in compression.
      */
-    template <typename Iterator>
+    template <typename Iterators>
     void
     reinit_cells(
-      const IteratorRange<Iterator> &             cell_iterator_range,
+      const Iterators &                           cell_iterator_range,
```

Comment:
The documentation of this function is now (possibly) out of sync with the argument list. In any case, it would be useful if you documented what kind of type `Iterators` can possibly be.

Same for the next function.